simcase-game/src/views/Ranking.js

```diff
@@ -2,27 +2,190 @@
+};
 
+const contentStyle = {
+  height: "75vh",
+  overflow: "auto",
+  color: "#4d4d4d",
+  textAlign: "center"
+};
+
+const tableStyle = {
+  width: "85%",
+  margin: "0 auto",
+  borderCollapse: "collapse",
+  fontSize: 16
+};
+
+const captionStyle = {
+  fontWeight: "bold",
+  fontSize: 18,
+  padding: 5
+};
+
+const col_1Style = {
+  width: "20%"
+};
+
+const col_2Style = {
+  // width: "50%"
+};
+
+const col_3Style = {
+  width: "30%"
+};
+
+const imgTableStyle = {
+  width: 40
+  // marginRight: 10,
+};
+
+const buttonOptions = {
+  textDecoration: "none",
+  color: "#FFFFFF",
+  backgroundColor: "#3477F2",
+  padding: "10px 15px",
+  margin: 4,
+  border: "none",
+  borderRadius: 5,
+  fontSize: "1em"
+};
+
+const buttonBarStyles = {
+  display: "flex",
+  flexDirection: "row",
+  justifyContent: "center",
+  alignItems: "center",
+  height: "10vh",
+  position: "absolute",
+  bottom: 0,
+  width: "100vw"
+};
+
+function Ranking(props) {
+  const players = leaderboardDummyData;
+  console.log("Datos pasados:", players);
+
+  console.log("Leaderboard Dummy Data:", Object.values(leaderboardDummyData));
   return (
-    <div>
-      <h1>Hello from Ranking</h1>
-      <Link css={buttonOptions} to="/game/1">
-        Play Again
-      </Link>
-      <Link css={buttonOptions} to="/">
-        Go Home
-      </Link>
-    </div>
+    <>
+      <Header />
+      <main css={contentStyle}>
+        <h2>Leaderboard</h2>
+        <section>
+          <table css={tableStyle}>
+            <caption css={captionStyle}>Top 5 players</caption>
+            <thead>
+              <tr>
+                <th
+                  scope="col"
+                  css={{
+                    ...col_1Style,
+                    backgroundColor: "#0C4785",
+                    color: "#FFFFFF",
+                    padding: 10
+                  }}
+                >
+                  #
+                </th>
+                <th
+                  scope="col"
+                  css={{
+                    ...col_2Style,
+                    backgroundColor: "#0C4785",
+                    color: "#FFFFFF"
+                  }}
+                >
+                  username
+                </th>
+                <th
+                  scope="col"
+                  css={{
+                    ...col_3Style,
+                    backgroundColor: "#0C4785",
+                    color: "#FFFFFF"
+                  }}
+                >
+                  Points
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              {players &&
+                Object.values(players).map((user, index) => {
+                  if (index < 5) {
+                    return (
+                      <tr key={user.username}>
+                        <td css={col_1Style}>{user.id}</td>
+                        <td
+                          css={{
+                            ...col_2Style,
+                            textAlign: "left",
```

Comment:
I think you could start working with this styles like you have in the top of the file creating consts